tools/cmake/mbed_greentea.cmake

```diff
@@ -43,42 +52,53 @@ macro(mbed_greentea_add_test)
         "${multipleValueArgs}"
         ${ARGN}
     )
-    add_subdirectory(${MBED_ROOT} build)
+    if(NOT "${MBED_GREENTEA_TEST_SKIPPED}" STREQUAL "")
+        set(msg "Skipping ${MBED_GREENTEA_TEST_NAME}: ${MBED_GREENTEA_TEST_SKIPPED}")
+        message("${msg}")
```

Comment:
Is `NOTICE` the correct log level for this message? We could set it to `DEBUG` potentially, as it's probably not that important for a user to see in most cases.

Reply:
I'll change it to debug. The custom target below echoes this message at build stage, which is enough for a user.

Reply:
Done